### PR TITLE
Add changelog for v0.8.0

### PR DESCRIPTION
**Pull request by rmarianski**
I only scanned through the commit log to come up with this list. It might be worthwhile to also review the prs that were applied for v0.8.0 to see if I missed anything.

@nvkelso could you review please?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+* Add inclusive pois
+* Include all z17-18 pois at z16
+* Return min_zoom from pois query
+* Move building addresses to z16
+* Move boundaries at z17 to z16
+* Include large hotels earlier
+* Make larger 'big box' stores appear at lower zooms.
+* Add spreadsheet based properties for sort_key
+* Update sort_key values across all layers
+* Move parking topside, and remove duplicate park
```
Comment (nvkelso) [inline CHANGELOG.md]: Don't need this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+* Fix regressions for `man_made`, `national_park`, `natural`
```
Comment (nvkelso) [inline CHANGELOG.md]: Don't need this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
+v0.8.0
+------
+* **Release date**: 2016-03-04.
+* Fixed migration loop to handle the case where there are no explicit migrations to run.
+* Railway route extraction no longer relies on the `planet_osm_nodes` table, which may be missing if flat nodes is enabled.
+* Add labels for bay water features
+* Add inclusive pois
```
Comment (nvkelso) [inline CHANGELOG.md]: I'll propose rewording this.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
+v0.8.0
+------
+* **Release date**: 2016-03-04.
+* Fixed migration loop to handle the case where there are no explicit migrations to run.
+* Railway route extraction no longer relies on the `planet_osm_nodes` table, which may be missing if flat nodes is enabled.
+* Add labels for bay water features
```
Comment (nvkelso) [inline CHANGELOG.md]: There were 3 total new types?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
+v0.8.0
+------
+* **Release date**: 2016-03-04.
+* Fixed migration loop to handle the case where there are no explicit migrations to run.
+* Railway route extraction no longer relies on the `planet_osm_nodes` table, which may be missing if flat nodes is enabled.
```
Comment (nvkelso) [inline CHANGELOG.md]: This has the benefit of including more data in the calculations, promoting the right stations to a better `tile_kind_rank`.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
...
+* Add labels for bay water features
+* Add inclusive pois
+* Include all z17-18 pois at z16
+* Return min_zoom from pois query
+* Move building addresses to z16
```
Comment (nvkelso) [inline CHANGELOG.md]: We also moved Fences to z16 (was z17).

For a different release, we should consider marking the addresses with `min_zoom:17`.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,26 @@
...
+* Move boundaries at z17 to z16
```
Comment (nvkelso) [inline CHANGELOG.md]: Let's say `fences`.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,26 @@
...
+* Include large hotels earlier
+* Make larger 'big box' stores appear at lower zooms.
```
Comment (nvkelso) [inline CHANGELOG.md]: We also added a couple new POI types as part of that work.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,26 @@
...
+* Add spreadsheet based properties for sort_key
+* Update sort_key values across all layers
+* Move parking topside, and remove duplicate park
+* Fix regressions for `man_made`, `national_park`, `natural`
+* Show all buildings at z16
+* Disable area/volume filter at z16
```
Comment (nvkelso) [inline CHANGELOG.md]: Was this for landuse or buildings or?

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,26 @@
...
+* Add min_zoom property to buildings
+* Update road brunnel, level sorting for z>=15
```
Comment (nvkelso) [inline CHANGELOG.md]: This was recalculated for all zooms.

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,26 @@
...
+* Add labels for bay water features
+* Add inclusive pois
+* Include all z17-18 pois at z16
+* Return min_zoom from pois query
+* Move building addresses to z16
+* Move boundaries at z17 to z16
+* Include large hotels earlier
+* Make larger 'big box' stores appear at lower zooms.
+* Add spreadsheet based properties for sort_key
+* Update sort_key values across all layers
```
Comment (nvkelso) [inline CHANGELOG.md]: We updated for `landuse` and `roads`, and added for many other layers.

File: CHANGELOG.md (hunk 6 above)
Comment (rmarianski) [inline CHANGELOG.md]: They actually are set that way already! We should update the wording then to make this clearer.

**Pull request merged**